frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -0,0 +1,303 @@
+                            RATE OF COMPLETION
+                        </h3>
+                        <span className="uppercase text-teal-4">
+                            Past 30 days
+                        </span>
+                    </div>
+                    <ResponsiveContainer width="100%" height={120}>
+                        <LineChart data={completionRateData}>
+                            <CartesianGrid strokeDasharray="3 3" />
+                            <XAxis dataKey="name" /> <YAxis domain={[0, 100]} />
+                            <Tooltip labelStyle={{ color: '#000' }} />
+                            <Line
+                                type="monotone"
+                                dataKey="rate"
+                                stroke="var(--teal-3)"
+                                strokeWidth={2}
+                                name="Completion Rate"
+                            />
+                        </LineChart>
+                    </ResponsiveContainer>
+                </div>
+            </div>
+
+            <div className="flex flex-wrap items-center justify-between gap-2 mb-4">
+                <div className="flex flex-row gap-x-2">
+                    <SearchBar
+                        searchTerm={searchTerm}
+                        changeCallback={handleChange}
+                    />
+                    <DropdownControl
+                        label="order by"
+                        setState={setSortQuery}
+                        enumType={{
+                            'Name (A-Z)': 'name_last asc',
+                            'Name (Z-A)': 'name_last desc',
+                            'Section Created (Newest)': 'created_at desc',
+                            'Section Created (Oldest)': 'created_at asc'
+                        }}
+                    />
+                    {/* pill  */}
+                    <button
+                        className={`px-4 py-2 rounded-full border ${
+                            includeActive
+                                ? 'bg-teal-600 text-white'
+                                : 'bg-white text-teal-600'
+                        }`}
+                        onClick={() => setIncludeActive(!includeActive)}
+                    >
+                        {includeActive
+                            ? 'Active Sessions'
+                            : 'Inactive Sessions'}
+                    </button>
+                </div>
+
+                <button className="button flex items-center">
+                    <PlusCircleIcon className="w-4 h-4 mr-1" />
+                    Add Section
+                </button>
+            </div>
+
+            {/* sections table */}
+            <div className="card p-4">
+                <table className="table w-full mb-4">
+                    <thead className="bg-background">
+                        <tr>
+                            <th>
+                                <input
```

Comment:
Can you make these checkboxes match the style we use elsewhere in the application?